src/assets/style/strip.css

```diff
@@ -1,108 +1,101 @@
-
 .strip {
-  background-color: rgba(255, 255, 255, 0.0);
-
-  border-left: 4px solid transparent;
-  border-right: 4px solid transparent;
-  border-top: 2px solid transparent; 
-  border-bottom: 2px solid transparent; 
-
-  list-style-type: none;
-  position: relative;
-  padding: 3px 10px;
-  margin-bottom: 5px;
-  margin-left: 0;
-  margin-right: -3px;
-
-  transition: margin-left 0.1s ease, margin-right 0.1s ease, background-color 0.05s ease, box-shadow 0.05s ease;
+    position: relative;
+    margin-bottom: 5px;
+    margin-left: 0;
+    margin-right: -3px;
+    padding: 3px 10px;
+    background-color: rgba(255, 255, 255, 0.0);
+    border-left: 4px solid transparent;
+    border-right: 4px solid transparent;
+    border-top: 2px solid transparent;
+    border-bottom: 2px solid transparent;
+    list-style-type: none;
+    transition: margin-left 0.1s ease,
+                margin-right 0.1s ease,
+                background-color 0.05s ease,
+                box-shadow 0.05s ease;
 }
 
 .strip:hover {
-  background-color: rgba(255, 255, 255, 0.1);
-  cursor: pointer;
+    background-color: rgba(255, 255, 255, 0.1);
+    cursor: pointer;
 }
 
 .strip.active {
-  background-color: rgba(255, 255, 255, 0.2);
-
-  box-shadow: 0 1px 7px rgba(0, 0, 0, 1);
-
-  margin-left: -5px;
-  margin-right: 5px;
+    margin-left: -5px;
+    margin-right: 5px;
+    background-color: rgba(255, 255, 255, 0.2);
+    box-shadow: 0 1px 7px rgba(0, 0, 0, 1);
 }
 
 .strip.arrival {
-  border-color: rgba(224, 128, 128, 0.8);
-  border-top-color: rgba(224, 128, 128, 0.1);
-  border-bottom-color: rgba(224, 128, 128, 0.1);
+    border-color: rgba(224, 128, 128, 0.8);
+    border-top-color: rgba(224, 128, 128, 0.1);
+    border-bottom-color: rgba(224, 128, 128, 0.1);
 }
 
 .strip.departure {
-  border-color: rgba(128, 255, 255, 0.8);
-  border-top-color: rgba(128, 255, 255, 0.05);
-  border-bottom-color: rgba(128, 255, 255, 0.05);
+    border-color: rgba(128, 255, 255, 0.8);
+    border-top-color: rgba(128, 255, 255, 0.05);
+    border-bottom-color: rgba(128, 255, 255, 0.05);
```

Comment:
short answer, no.

long answer; if we were using postCSS we could.  but I'm not really a fan of postCSS and there is a fair amount of setup (and it's overkill here)